@re-/assert/test/metaTests/testHelpers.ts

```diff
@@ -0,0 +1,71 @@
+import { copyFileSync } from "node:fs"
+import { fromHere, readJson, shell } from "@re-/node"
+import { ExpressionStatement, Project, SourceFile, SyntaxKind } from "ts-morph"
+
+export type TestData = {
+    statements: ExpressionStatement[]
+    expected: Record<number | string, string>
+    sourceFile: SourceFile
+    initialText: string
+}
+const snapMatch = /\.snap\((.+)?\)/
+const benchMeanMedianMatch = /\.(mean|median)\(`\d+\.\d+(ns|us|s|ms)`\)/
```

Comment:
You can probably tell by the number of times the individual strings `mean|median` or `ns|us|ms|s` appear that there's some potential refactoring that would simplify maintenance if we were to add new time units or measures in the future.

One approach would be to use something like this:

```ts
const BENCH_STATS = ["mean", "median"]
const TIME_UNITS = ["ns", "us", "ms", "s"]

// This is just an example and definitely won't work as written KEKW
const createBenchStatExpression = (asMark: boolean) => `${BENCH_STATS.join("|")}${asMark ? ":" : "("}${TIME_UNITS.join("|")}${asMark ? ",?" : ")"}`
// Unlike regex literals, the RegExp class can be created dynamically and use template strings
const benchStatSnapCall = new RegExp(createBenchStatExpression(false))
const benchMarkSnapCall = new RegExp(`\.mark...${createBenchStatExpression(true)}+`)
```

Let's also make sure that all of the functionality related to bench regex lives here as opposed to also having references to similar expressions in `metaBench.test.ts`.